src/instantiate.ts

```diff
@@ -245,7 +245,7 @@ export async function loadAllofExtension(context: vscode.ExtensionContext) {
           }
 
           if (canRun && [`member`, `streamfile`, `file`].includes(uri.scheme)) {
-            CompileTools.runAction(instance, uri);
+            CompileTools.runAction(instance, uri, action, method);
```

Comment:
We should `await` here if the caller of the command want to actually wait for it to end.
```suggestion
            await CompileTools.runAction(instance, uri, action, method);
```

Reply:
Good point. Update that: cf3a4a5